lk2nd/device/dts/msm8953/msm8953-asus-zenfone3.dts

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8953 0>;
+	qcom,board-id = <21 0>;
+};
+
+&lk2nd {
+	model = "Asus Zenfone 3";
+	compatible = "asus,zenfone3";
+	lk2nd,dtb-files = "msm8953-asus-zenfone3", "ze552kl-sr1_msm8953-mtp";
```

Comment:
```suggestion
	lk2nd,dtb-files = "msm8953-asus-zenfone3";
```

This is only useful for mainline, and if someone ever decided to reuse this tooling for downstream they're better just renaming the dtb file since for some devices the file name would clash anyway, when oem just modified the mtp/qrd/whatever file...